GNUmakefile

```diff
@@ -549,21 +548,17 @@ omptarget_src := \
         src/omptarget/device_tzset.cc \
         # End. Add alphabetically.
 
-omptarget_hdr := \
-        src/omptarget/device_util.hh
```

Comment:
Maybe I missed it but I do not see this file being deleted.

Reply:
It isn't, just `${omptarget_hdr}` is never referenced in the GNUmakefile, so the variable isn't needed. Whereas we need `${cuda_hdr}` and `${hip_hdr}` because we generate the HIP header from the CUDA header.